pom.xml

```diff
@@ -452,7 +452,7 @@
     <version.junit>4.11</version.junit>
     <version.org.jboss.byteman>2.1.3</version.org.jboss.byteman>
     <version.org.jboss.arquillian.core>1.1.2.Final-wildfly-1</version.org.jboss.arquillian.core>
-    <version.org.jboss.jboss-transaction-spi>7.1.0.Final</version.org.jboss.jboss-transaction-spi>
+    <version.org.jboss.jboss-transaction-spi>7.1.1.Final-SNAPSHOT</version.org.jboss.jboss-transaction-spi>
```

Comment:
You may as well attribute it to yourself.

git reset HEAD~1
git add .
git commit -m "Mikes magic"

Probably best to do this on a test branch first :)